### PR TITLE
UKIS content changes

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Changes described in https://trello.com/c/i4bB9SEk/768-ukis-content-changes

### How to review 
Are the changes correct?

### CODE REVIEW

File: app/themes/ukis/templates/partials/header.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<header class="header u-mb-s">
+    <div class="header__top" role="banner">
+        <div class="container">
+            <div class="grid grid--gutterless">
+                <div class="grid__col col-6@s">
```
Comment (hamishtaplin) [inline app/themes/ukis/templates/partials/header.html]: Make logo smaller.

```suggestion
                <div class="grid__col col-7@xs col-6@s">
```

**Review by hamishtaplin — CHANGES_REQUESTED** [review]

**Review by hamishtaplin — APPROVED** [review]

**Review by tgandrews — APPROVED** [review]
I haven't run it but the changes LGTM

**Comment by jonnyshaw89** [comment]
:man_facepalming:

**Review by samiwel — APPROVED** [review]
LGTM

**Pull request merged**